src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -3233,41 +3233,6 @@ public static int Main()
                 Diagnostic(ErrorCode.ERR_AddOrRemoveExpected, "value"));
         }
 
-        [WorkItem(536956, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/536956")]
-        [Fact]
-        public void CS1065ERR_DefaultValueNotAllowed()
```

Comment:
>CS1065ERR_DefaultValueNotAllowed [](http://example.com/codeflow?start=20&length=32)

Unless there's a good reason, we should keep existing tests, even if the behavior has changed, so that we're confirming the current expected behavior.